components/search/TierToggler.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <nuxt-link
+        :to="$route.fullPath"
```

Comment:
the link will need to lead to the toggled version not the current path.

Reply:
this should now be set up correctly